### PR TITLE
[16.0][MIG] partner_contact_access_link: Migration to 16.0

### PR DESCRIPTION
**Pull request by vib-adhoc**

### CODE REVIEW

File: partner_contact_access_link/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    _inherit = "res.partner"
+
+    def open_child_form(self):
+        """Open child contact form from the parent partner form view"""
+        return {
+            "type": "ir.actions.act_window",
+            "res_model": "res.partner",
+            "res_id": self.id,
+            "view_mode": "form",
+            "view_type": "form",
```
Comment (legalsylvain) [inline partner_contact_access_link/models/res_partner.py]: Afaik view_type is deprecated un V16.

**Review by legalsylvain — APPROVED** [review]
A minor remarks. Lgtm otherwise.

File: partner_contact_access_link/models/res_partner.py (hunk 1 above)
Comment (vib-adhoc) [reply partner_contact_access_link/models/res_partner.py]: Thanks!

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Pull request merged**